.appveyor.yml

```diff
@@ -0,0 +1,42 @@
+version: '0.6.7.{build}'
```

Comment:
It has been increased as part of the release.  But in reality it doesn't really matter since it's not used for anything outside of appveyor.